helpers/renderCopyButtons.ts

```diff
@@ -0,0 +1,145 @@
+import { RefObject } from "react";
+type Target = HTMLButtonElement | HTMLSpanElement | HTMLImageElement;
+
+const onClickCheck = (event: MouseEvent): void => {
+  const target = event.target;
+
+  const targetClasslistCheck = (target: Target) => {
+    const buttonCopyClasses = ["btn-copy", "btn-copy-img", "btn-copy-text"];
+
+    if (buttonCopyClasses.includes(target.className)) {
+      return true;
+    }
+  };
+
+  const areChildNodesAndClassesValid = (
+    buttonImg: ChildNode,
+    buttonText: ChildNode,
+    target: Target
+  ): buttonImg is HTMLElement => {
+    if (
+      buttonImg instanceof HTMLElement &&
+      buttonText &&
+      targetClasslistCheck(target)
```

Comment:
```suggestion
  buttonCopyClasses.includes(target.className)
```

And I think we can remove this method `targetClasslistCheck`

Only keep `buttonCopyClasses`
@ntorbinskiy